### PR TITLE
Working on the Generators

### PR DESCRIPTION
**Pull request by webhead1104**

### CODE REVIEW

File: build.gradle.kts
Comment (Foxikle) [inline build.gradle.kts:1]: We agreed on `plugin_dir`

File: src/main/java/dev/foxikle/webnetbedwars/managers/GameManager.java
Comment (Foxikle) [inline src/main/java/dev/foxikle/webnetbedwars/managers/GameManager.java:1]: Looks good :+1:

File: src/main/java/dev/foxikle/webnetbedwars/managers/GeneratorManager.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        for (Team t : plugin.getGameManager().getTeamlist()) {
+//iron
+            Bukkit.getScheduler().runTaskTimer(plugin, () -> world.dropItemNaturally(t.generatorLocation(), Items.IRON), 0, plugin.getConfig().getInt("GeneratorsWaitTime.iron"));
+//gold
+            Bukkit.getScheduler().runTaskTimer(plugin, () -> world.dropItemNaturally(t.generatorLocation(), Items.GOLD), 0, plugin.getConfig().getInt("GeneratorsWaitTime.gold"));
+        }
+
+
+
```
Comment (Foxikle) [inline src/main/java/dev/foxikle/webnetbedwars/managers/GeneratorManager.java:32]: Don't need these spaces

**Review by Foxikle — CHANGES_REQUESTED** [review]
Just fix the `build.gradle.kts` :)

**Review by Foxikle — APPROVED** [review]
Looks good :+1:

File: src/main/java/dev/foxikle/webnetbedwars/managers/GeneratorManager.java (hunk 1 above)
Comment (webhead1104) [reply src/main/java/dev/foxikle/webnetbedwars/managers/GeneratorManager.java:32]: i will fix those now

**Pull request merged**